src/axes.jl

```diff
@@ -366,18 +366,20 @@ end
 # -------------------------------------------------------------------------
 
 # push the limits out slightly
-function widen(lmin, lmax)
-    span = lmax - lmin
+function widen(lmin, lmax, scale)
+    sf = scalefunc(scale)
+    isf = invscalefunc(scale)
+    span = sf(lmax) - sf(lmin)
     # eps = max(1e-16, min(1e-2span, 1e-10))
     eps = max(1e-16, 0.03span)
-    lmin-eps, lmax+eps
+    isf(lmin-eps), isf(lmax+eps)
 end
 
 # figure out if widening is a good idea.  if there's a scale set it's too tricky,
 # so lazy out and don't widen
 function default_should_widen(axis::Axis)
     should_widen = false
-    if axis[:scale] == :identity && !is_2tuple(axis[:lims])
+    if (!(axis[:scale] in _logScales && axis[:extrema].emin <= 0)) && !is_2tuple(axis[:lims])
```

Comment:
Does that mean that widening doesn't happen for `axis[:scale] == :identity` any longer (I don't quite know what widening does, I have to admit).